api/configs/middleware/__init__.py

```diff
@@ -203,6 +204,7 @@ class MiddlewareConfig(
     GoogleCloudStorageConfig,
     TencentCloudCOSStorageConfig,
     HuaweiCloudOBSStorageConfig,
+    BaiduOBSStorageConfig,
```

Comment:
How about place it in the place after `AzureBlobStorageConfig` maintaining the alphabet order?

Reply:
done